EDITORS.rst

```diff
@@ -0,0 +1,35 @@
+Welcome
+=======
+
+Thank you for helping edit the NixOS "Weekly" Newsletter!
+
```

Comment:
I would add a section about the requirements needed to be able to perform an editor task. Currently this is the list I have (might be incomplete):

- Git (dah!)
- nix
- push permission to nixos/nixos-weekly production branch
- mailchimp credentials account to send newsletter.